### PR TITLE
Update VirtualResolutions package to v1.2

### PR DESCRIPTION
**Pull request by mahrud**

### CODE REVIEW

File: M2/Macaulay2/packages/VirtualResolutions.m2 [hunk 1]
```diff
@@ -556,6 +561,6 @@ uninstallPackage "VirtualResolutions"
 restart
 installPackage "VirtualResolutions"
 restart
-needsPackage("VirtualResolutions", FileName => "./VirtualResolutions.m2")
+needsPackage("VirtualResolutions", FileName => "VirtualResolutions.m2")
```
Comment (DanGrayson) [inline M2/Macaulay2/packages/VirtualResolutions.m2:564]: Why is the FileName option still needed?

File: M2/Macaulay2/packages/VirtualResolutions/tests.m2 [hunk 2]
```diff
@@ -200,6 +211,7 @@ TEST ///
     assert(multigradedRegularity(S, I) == {{1,5},{2,2},{4,1}})
 ///
 
+-- This test currently fails due to a bug in tateResolutions
 ///
```
Comment (mahrud) [inline M2/Macaulay2/packages/VirtualResolutions/tests.m2:215]: Ahh, this is fixed in TateOnProducts now. @DanGrayson could you remove the comment and enable that test in a commit to development? I'd rather not make a new PR for one line.

File: M2/Macaulay2/packages/VirtualResolutions.m2 (hunk 1 above)
Comment (mahrud) [reply M2/Macaulay2/packages/VirtualResolutions.m2:564]: I'm not sure who needed that, but it's in the footer, so does it matter?

File: M2/Macaulay2/packages/VirtualResolutions/tests.m2 (hunk 2 above)
Comment (DanGrayson) [reply M2/Macaulay2/packages/VirtualResolutions/tests.m2:215]: okay  :  commit 6cbb03398cd607181c78ab1c69663a7e94abac64

**Pull request merged**